src/python/pants/backend/graph_info/tasks/dependees.py

```diff
@@ -31,7 +31,7 @@ def register_options(cls, register):
             "--transitive",
             type=bool,
             default=False,
-            help="Whether to include only the first level of dependendees.",
```

Comment:
This should be `dependees`.

Reply:
Out of curiosity, is this similar to `dependents`?

(If this is an API term, then obviously you can't change it...)

Reply:
Yea, they're synonyms afaik.

Reply:
If you're ok, I'd be happy to write a patch to fix my error above to `dependent` and change and  `dependee` in general to `dependent`.

Reply:
That would be a non-trivial patch. I think that given that `dependee` is used in this context: https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=dependee&btnG= (albeit much less frequently than `dependent`) it's probably fine to leave it.

Reply:
Agreed with sticking with `dependee`. It is a bit weird, but it is valid in a technical context like this and we've used this name for several years.

Reply:
At 1/1000, I'd treat that whole set as a typo :-)

That said, my goal is to only do work that will be accepted. My tooling usually doesn't mind changing lots of files (admittedly having to reformat files is a bit more work), and obviously the larger the change the more stress for reviewers.